extensions/wikia/Search/WikiaSearchAjaxController.class.php

```diff
@@ -27,6 +27,7 @@ public function getNextResults() {
 		$crossWikia = $this->request->getBool( 'crossWikia' );
 		$hub = false;
 
+		// TODO: refactor to avoid using wgEnableWikiaHomePageExt
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)